### PR TITLE
Adding basic flow

### PR DESCRIPTION
**Pull request by delekta**
Creating:
- admin panel (forms for adding events)
- events screen
- event details where you can buy tickets
 

https://user-images.githubusercontent.com/57197926/235731312-ba12cd21-4db8-4244-afe6-904d7b41709c.mov

### CODE REVIEW

File: frontend/src/components/EventCard.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        event: props.event,
+      },
+    });
+  };
+
+  return (
+    <VerticalContainer onClick={onClick}>
+      <StyledName>{props.event.name}</StyledName>
+      <StyledDate>
+        Date: <br></br>
```
Comment (t0ms0n00) [inline frontend/src/components/EventCard.tsx:26]: Could you use self-closing tag?

File: frontend/src/screens/AdminPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          required
+        />
+        <LabelText htmlFor="eventTicketsNumber"> Number of tickets </LabelText>
+        <StyledInput
+          type="number"
+          className="form-control"
+          {...register("eventTicketsNumber")}
+          required
+        />
+        <StyledButton type="submit">Save</StyledButton>
```
Comment (t0ms0n00) [inline frontend/src/screens/AdminPanel.tsx:71]: Wdyt about changing cursor to pointer here?

File: frontend/src/screens/AdminPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        />
+        <LabelText htmlFor="eventDate"> Event date </LabelText>
+        <StyledInput
+          type="date"
+          className="form-control"
+          {...register("eventDate")}
+          required
+        />
+        <LabelText htmlFor="eventTicketsNumber"> Number of tickets </LabelText>
+        <StyledInput
```
Comment (t0ms0n00) [inline frontend/src/screens/AdminPanel.tsx:65]: I can set a negative value here without any error message. Or it was done like that on purpose assuming that we will validate this on the contract side?

**Review by t0ms0n00 — APPROVED** [review]
Awesome job! The app looks really nice now 🚀 . Just small suggestions from me, but nothing crucial, so for me you can merge this without applying them.

File: frontend/src/screens/AdminPanel.tsx (hunk 3 above)
Comment (delekta) [reply frontend/src/screens/AdminPanel.tsx:65]: For now, the submit button is disabled if number of tickets is negative 
```
        <StyledButton disabled={ticketsNumber <= 0} onClick={buyTicket}>
          Buy tickets
        </StyledButton>
```
We can work on legit error message later

**Pull request merged**